subiquity/tests/api/test_api.py

```diff
@@ -1230,6 +1230,111 @@ async def test_osprober_knames(self):
             }
             self.assertEqual(expected, nvme_p2['os'])
 
+    @timeout()
+    async def test_edit_should_trigger_wipe_when_requested(self):
+        # LP: #1983036 - a partition wipe was requested but didn't happen
+        # 1) the wipe value was ignored, as the format key was used instead
+        # 2) the format wasn't actually changed, so the partition was
+        #    considered preserve=True
```

Comment:
I think this comment is out of date now?

Reply:
Updated, thanks.